group_vars/matrix_servers

```diff
@@ -1329,6 +1329,34 @@ matrix_backup_borg_systemd_required_services_list: |
 # /matrix-backup-borg
 #
 ######################################################################
+######################################################################
+#
+# matrix-cactus-comments
+#
+######################################################################
+
+matrix_cactus_comments_enabled: false
+
+matrix_cactus_comments_container_image_self_build: "{{ matrix_architecture not in ['amd64', 'arm32', 'arm64'] }}"
+
+# Normally, matrix-nginx-proxy is enabled and nginx can reach cactus-comments over the container network.
+# If matrix-nginx-proxy is not enabled, or you otherwise have a need for it, you can expose
+# cactus-comments web-server port.
+#matrix_cactus_comments_container_http_host_bind_port: "{{ '' if matrix_nginx_proxy_enabled else '127.0.0.1:' + matrix_cactus_comments_container_port | string }}"
+
+matrix_cactus_comments_systemd_required_services_list: |
+  {{
+    (['docker.service'])
+    +
+    (['matrix-' + matrix_homeserver_implementation + '.service'])
```

Comment:
```suggestion
    +
    (['matrix-' + matrix_homeserver_implementation + '.service'])
    +
    (['matrix-nginx-proxy.service'] if matrix_nginx_proxy_enabled else [])
```

Services usually acccess the homeserver through the nginx proxy, and not directly. Adding it as a dependency makes sense.

In this case, `matrix_cactus_comments_homeserver_url` is set to `matrix_homeserver_url` which is the public URL (`https://matrix.DOMAIN`). That may need to be redone as well using `matrix_homeserver_container_url`.

Reply:
done